JavaScript/hacker-news/public/js/main.js

```diff
@@ -0,0 +1,66 @@
+// this API Key will only work on Algorithmia's website; get your own key at https://algorithmia.com/user#credentials
+var client = Algorithmia.client('sim3fGsmXfk6P0o1Kx8rzo2ZmAf1');
```

Comment:
:bulb: Let's add a quick comment here re/ where this key lives and its permissions.

Reply:
As noted below, added this to the tracking sheet